server/tests/factory/userFactory.js

```diff
@@ -0,0 +1,24 @@
+import faker from 'faker';
```

Comment:
I believe there's already a `user-factory.js` file that does this

Reply:
also, you could modify the `createTestUser` factory to make a call to  `createProfileDetails` then have another function that would be something like `createTestUserWithoutProfile`.